kd/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: pttg-ip-smoke-tests
+  labels:
+    version: {{.VERSION}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: pttg-ip-smoke-tests
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: pttg-ip-smoke-tests
+        version: {{.VERSION}}
+    spec:
+      imagePullSecrets:
+        - name: registrykey
+      containers:
+        - name: pttg-ip-smoke-tests
+          image: quay.io/ukhomeofficedigital/pttg-ip-smoke-tests:latest
```

Comment:
Usual comment about use of "latest". Might get a situation where there are new and merged smoke tests that will only work with an, as yet, unavailable deployment.

Reply:
Addressed